extensions/sql-migration/src/dialog/migrationCutover/migrationCutoverDialogModel.ts

```diff
@@ -112,6 +112,7 @@ export class MigrationCutoverDialogModel {
 		} catch (error) {
 			this.CancelMigrationError = error;
 			console.log(error);
```

Comment:
Would probably suggest adding the console.log to `logError` so that you can still see stuff in the console logs if an error occurs

Reply:
You can remove this console.log now